service/resource/configmap/configmap.go

```diff
@@ -36,6 +39,7 @@ func DefaultConfig() Config {
 		ConfigMapKey:         "",
 		ConfigMapName:        "",
 		ConfigMapNamespace:   "",
+		ReloadWaitTime:       time.Duration(0),
```

Comment:
You can simply use untyped `0`. You already check against untyped 0 below `config.ReloadWaitTime == 0`.